python.properties

```diff
@@ -35,4 +35,4 @@ python.cachedir.skip=False
 # you may need to also ensure that jmri_bindings.py is executed at application
 # start.
 # Note that the first character of true or false needs to be lowercase.
```

Comment:
The jython.exec property is **not** passed onto Jython, but is consumed by Java. Note the last line in the comments.